exam_builder/app/app.js

```diff
@@ -11,7 +11,6 @@ App = Ember.Application.extend({
   modulePrefix: config.modulePrefix,
   podModulePrefix: config.podModulePrefix,
   Resolver: Resolver,
-  rootElement: '#exam_builder',
```

Comment:
* I assume your production `app/views/layouts/application.html` is more
  involved than what you've provided here, and probably includes the
  `#exam_builder` element. Unfortunately, for the sake of reproducing, I
  had to remove the `rootElement` configuration from
  `exam_builder/app/app.js`.